backend/app/services/adapters/executor_job.py

```
@@ -49,143 +72,424 @@ def cleanup_task_executor(
+                    [
+                        SubtaskStatus.PENDING,
+                        SubtaskStatus.COMPLETED,
+                        SubtaskStatus.FAILED,
+                        SubtaskStatus.CANCELLED,
+                    ]
+                ),
+                Subtask.created_at > lookback_start,
+                Subtask.created_at <= cutoff,
+                Subtask.executor_name.isnot(None),
+                Subtask.executor_name != "",
+                Subtask.executor_deleted_at == False,
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `False`; use `not Subtask.executor_deleted_at:` for false checks

Replace with `not Subtask.executor_deleted_at`

(E712)